packages/yourloops/pages/hcp/patients/models.ts

```diff
@@ -37,14 +37,16 @@ export interface PatientListProps {
   onClickPatient: (user: TeamUser) => void;
   onFlagPatient: (userId: string) => Promise<void>;
   onSortList: (field: SortFields, direction: SortDirection) => void;
+  onClickRemovePatient: (patient: TeamUser) => void;
 }
 
 export interface PatientElementProps {
-  trNA: string;
+  trNA?: string;
```

Comment:
The purpose of this field was mandatory, was to avoid re-fetch this translation for every cars/row, since there is a lots of them.
It should either kept that way, or completely removed.